utils.js

```diff
@@ -10,22 +10,22 @@ import * as Main from 'resource:///org/gnome/shell/ui/main.js';
 import * as Ripples from 'resource:///org/gnome/shell/ui/ripples.js';
 import * as Config from 'resource:///org/gnome/shell/misc/config.js';
 
-import { Lib } from './imports.js';
+import { Utils, Lib } from './imports.js';
```

Comment:
wait you can do that??? just let the file circular-import itself??

Reply:
and it works flawlessly. Wow uh, JS I guess :sweat_smile:

Reply:
Oops, yeah, you can do that fine... although I didn't even realise it's utils.js itself (lol) - so don't need to import.  I fixup.

Reply:
I think you do need to import, when I tried using the class on top it didn't work so there's still an order scope a la C/C++

Reply:
@Thesola10, is this still an issue?  happy to revert that change if it's causing issues (e.g. use the import).  There is a order to modules being enabled that can cause issues).

Let me know.

Reply:
@jtaala Works now, hence why I marked this convo as resolved!

i'm just not used to JS' way of doing things I guess

Reply:
Okay, I'll merge this then.  Thanks @Thesola10.